homeassistant/components/fido/sensor.py

```diff
@@ -97,32 +182,18 @@ def __init__(self, fido_data, sensor_type, name, number):
         self.client_name = name
         self._number = number
         self.type = sensor_type
```

Comment:
This can be removed, as there is now `self.entity_description.key`

Reply:
I do not think so, e.g.
name = config.get(CONF_NAME)
they are other values, than what are stored in entity_description.